app/tests/unit/components/Form/ProjectMilestoneReportFormSummary.test.tsx

```diff
@@ -141,5 +148,258 @@ describe("The Project Milestone Report Form Summary", () => {
+      screen.getByText("Net Payment Amount This Milestone")
+    ).toBeInTheDocument();
+    expect(screen.getByText(/\$888\.00/i)).toBeInTheDocument();
+    expect(
+      screen.getByText("Holdback Amount This Milestone")
+    ).toBeInTheDocument();
+    expect(screen.getByText(/\$111\.00/i)).toBeInTheDocument();
+
+    // adjusted values
+    expect(
+      screen.getByText("Gross Payment Amount This Milestone (Adjusted)")
+    ).toBeInTheDocument();
+    expect(screen.getByText(/\$99\.00/i)).toBeInTheDocument();
+    expect(
+      screen.getByText("Net Payment Amount This Milestone (Adjusted)")
+    ).toBeInTheDocument();
+    expect(screen.getByText(/\$11\.00/i)).toBeInTheDocument();
+    expect(
+      screen.getByText("Holdback Amount This Milestone (Adjusted)")
+    ).toBeInTheDocument();
+    expect(screen.getByText(/\$88\.00/i)).toBeInTheDocument();
+  });
+});
+
+const latestCommittedData = {
+  latestCommittedMilestoneFormChanges: {
+    edges: [
+      {
+        node: {
+          newFormData: {
+            totalEligibleExpenses: 1000,
+            description: "charmander",
+            projectId: 1,
+            reportingRequirementIndex: 1,
+            reportType: "General",
+            reportDueDate: "2020-01-10T23:59:59.999-07:00",
+            reportingRequirementId: 1,
+            hasExpenses: true,
+            calculatedGrossAmount: 567,
+            calculatedNetAmount: 789,
+            calculatedHoldbackAmount: 891,
+            adjustedNetAmount: 89,
+            adjustedGrossAmount: 67,
+            adjustedHoldBackAmount: 91,
+          },
+        },
+      },
+    ],
+  },
+};
+
+const modifiedMockQueryPayload = {
+  ...mockQueryPayload,
+  ProjectRevision() {
+    const originalProjectRevision = mockQueryPayload.ProjectRevision();
+    const modifiedProjectRevision = {
+      ...originalProjectRevision,
+      latestCommittedMilestoneFormChanges: {
+        edges: [
+          {
+            node: {
+              newFormData:
+                latestCommittedData.latestCommittedMilestoneFormChanges.edges[0]
+                  .node.newFormData,
+            },
+          },
+        ],
+      },
+    };
+    return modifiedProjectRevision;
+  },
+};
+
+describe("Project Milestone Summary with latest committed data", () => {
+  it("Displays diffs of the data fields that were updated and shows latest committed values", () => {
+    const componentTestingHelperWithLatestCommittedData =
+      new ComponentTestingHelper<ProjectMilestoneReportFormSummaryQuery>({
+        component: ProjectMilestoneReportFormSummary,
+        testQuery: testQuery,
+        compiledQuery: compiledProjectMilestoneReportFormSummaryQuery,
+        getPropsFromTestQuery: (data) => ({
+          query: data.query,
+          projectRevision: data.query.projectRevision,
+        }),
+        defaultQueryResolver: modifiedMockQueryPayload,
+        defaultQueryVariables: {},
+        defaultComponentProps: defaultComponentProps,
+      });
```

Comment:
Do we need a new describe block and component testing helper here? I think it could work to `loadQuery({...new mock here...})

Reply:
I thought it would be easier to read, we don't explicitly need it.